### ISSUE TITLE
Correct typos in README under Hands on with Handlebars Lab

### ISSUE THREAD

**Issue by BenGitsCode** [ga-wdi-boston/handlebars#25]
```diff
- Make sure to note any questions you come acorss and we'll go over them as a class.
+ Make sure to note any questions you come across and we'll go over them as a class.
```

**Comment by BenGitsCode** [comment]
There are several small typos in this section. I will include them all here as diffs as to not open several issues for small typos that are all isolated in that one section. 

```diff
- Lab: Hands on with Handlebars
+ Lab: Hands-on with Handlebars
```
Hands-on should be hyphenated. 

```diff
- Be able to explain in plain english what is happening.
+ Be able to explain in plain English what is happening.
```
English should be capitalized. 

```diff
- Let's look through the documentation and see if there is anyway we can improve this code.
+ Let's look through the documentation and see if there is any way we can improve this code.
```

Should be any way, not anyway.

**Comment by jrhorn424** [comment]
I just corrected a typo in your issue title. :wink:
